### PR TITLE
merge MFOS docs to master

### PR DESCRIPTION
**Pull request by sarahwiley**

### CODE REVIEW

File: docs/publish.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+#### Upload Content to Swarm
+
+By default, a vault is configured to connect to the [Swarm gateways](https://swarm-gateways.net/) to interact with Swarm. This works well for downloads and other requests, however uploads are limited to 1 MB, which might prevent from being able to upload some apps.
+In order to upload apps contents larger than 1 MB, a local Swarm node must be used. Installation instructions are available in the [Swarm guide](https://swarm-guide.readthedocs.io/en/latest/installation.html) and a [Docker image](https://github.com/ethersphere/swarm-docker) is also available.
+
+Your vault settings will need to be updated to use the local node rather than the Swarm gateways, running the following command (replace `http://localhost:8500` by the Swarm URL if different):
+
+```
+./packages/cli/bin/run vault:settings --bzz-url http://localhost:8500
```
Comment (PaulLeCam) [inline docs/publish.md]: Do we still need to document this? I thought we set the default Swarm URL to our own gateway, without upload limitation?

File: website/README.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
+This website was created with [Docusaurus](https://docusaurus.io/).
```
Comment (PaulLeCam) [inline website/README.md]: This doc should be removed or at least edited I think.

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,10 @@
+FROM node:8.11.4
```
[the remark this answers is not in the document]
Comment (sarahwiley) [reply Dockerfile]: docusaurus automatically sets up a docker configuration as an alternative process for running/serving/setting up your docs site. I'll remove it.

File: docs/introduction.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+### Mainframe OS Setup
+Follow onscreen instructions to secure your vault, setup your identity, and create a new software wallet (or import/connect an existing wallet).
+
+### Add a Contact
+
+**Mainframe OS > Contacts > Add Contact**
+Paste Contact ID of another Mainframe OS User
+
+Jake: `a5d66aa829306a04594634b451ca03f293a115a8f37c558f660c2db5bb5bdc83`
```
[the remark this answers is not in the document]
Comment (sarahwiley) [reply docs/introduction.md]: fixed.

File: docs/introduction.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+#### Create a developer identity
+ * choose a name for the identity that you will use to publish dapps under.
+ * select "create."
+
+Once you have created your Developer Identity, click **Add** to
+
+#### Import your dapp
+
+ * use "test" for the name.
+ * 0.0.1 for the version.
```
[the remark this answers is not in the document]
Comment (sarahwiley) [reply docs/introduction.md]: fixed.

File: docs/publish.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
+---
+id: publish
+title: Publish Your Dapp
+---
+
+### Packaging dapps for Mainframe OS
+
+Mainframe dapps must be packaged as static Web apps with an `index.html` entry file. All assets reference paths must be relative rather than rely on a specific resolution behaviour.
+For example to add the [Onyx Stats](applications/onyx-stats) contents to the Launcher, the `dist` folder (created during the build) must be selected rather than the root application folder.
```
[the remark this answers is not in the document]
Comment (sarahwiley) [reply docs/publish.md]: fixed.

File: docs/publish.md (hunk 1 above)
Comment (sarahwiley) [reply docs/publish.md]: fixed.

File: website/README.md (hunk 2 above)
Comment (sarahwiley) [reply website/README.md]: removed.

**Pull request merged**